README.md

```diff
@@ -1,50 +1,129 @@
 
-```
-SPACECRAFT_GCP_PROJECT=...
-SPACECRAFT_GCP_LOCATION=...
-SPACECRAFT_GCP_ZONE=...
-GOOGLE_APPLICATION_CREDENTIALS=...
-```
+Cloudflare is used for creating domain name records for every new network. the records point to various services deployed in kubernetes engine. Our cloudflare account is already configured with [spacemesh.io](http://spacemesh.io) nameservers therefore spacecraft can create the sub-domain records in it. Spacecraft currently doesn't allow us to provide root domain name as CLI option. It has spacemesh.io root domain hardcoded. 
 
-To learn more on how to set the `GOOGLE_APPLICATION_CREDENTIALS` variable refer this link https://cloud.google.com/docs/authentication/getting-started
+## ENVs and Secrets
 
-`SPACECRAFT_GCP_PROJECT` is the project id which you want to use for this tool and `SPACECRAFT_GCP_LOCATION`/`SPACECRAFT_GCP_ZONE` indicates the location and zone respectively for the GKE cluster.
+To use spacecraft you need to have the following ENVs set:
 
-### Clone and Build CLI
+```jsx
+# GCP Authentication
 
-Clone the repository and build the CLI binary. Run the below commands to do that:
+SPACECRAFT_GCP_PROJECT=
+SPACECRAFT_GCP_LOCATION=
+SPACECRAFT_GCP_ZONE=
+GOOGLE_APPLICATION_CREDENTIALS=
 
+# Cloudflare Authentication
+
+SPACECRAFT_CLOUDFLARE_API_TOKEN=
+
+# Slack Authentication
+
+SPACECRAFT_SLACK_CHANNEL_ID=
+SPACECRAFT_SLACK_TOKEN=
+
+# SSL Cert and Key
+
+SPACECRAFT_TLS_CRT=
+SPACECRAFT_TLS_KEY=
 ```
-git clone git@github.com:spacemeshos/go-spacecraft.git
-cd go-spacecraft
-go build
-```
 
-### Create the network
+If you don't want slack alerts or if you don't want to deploy web services then you can omit the slack and SSL ENVs respectively. 
+
+> All the secrets are stored in 1password and tagged as go-spacecraft.
+> 
+
+## Using the CLI
 
-The final step is to create the network. Run the below command to create a devnet:
+You can download and compile spacecraft using the following commands:
 
+```jsx
+git clone git@github.com:spacemeshos/go-spacecraft.git
+cd go-spacecraft && go build
 ```
-./go-spacecraft createNetwork --config=./artifacts/devnet/config.json
+
+Now you can explore the CLI flags and sub-commands using:
+
+```jsx
+./go-spacecraft --help
 ```
 
-This command will take around ~10min to finish and at the end you should see the kibana URL. The generated config file is uploaded to GCP storage. 
+Now you can refer to [devnet deployment guide](docs/devnet.md) to understand how to use the CLI to deploy a new network and web services.
+
+## Network Deployment
+
+For creating a new network boostrap deploys one bootstrap node, multiple bootnodes and then multiple miners. The bootnodes connect to bootstrap for p2p discovery and miners connect to bootnodes for p2p discovery. 
```

Comment:
Try to give more details here, to make it clearer.

Reply:
I am not sure what else to be added here.